app/send-mail.js

```diff
@@ -19,37 +19,45 @@ var transporter = nodemailer.createTransport({
         rejectUnauthorized: false
     }
 });
-exports.sendMail = function sendMailToGmail(subject, to_mail_list, mail_body, report_title, share_link, build_url, dash_board, imagefilename) {
```

Comment:
if we do not need to send mail to certain chart...it has to be handled in API itself.....